tests/src/main/java/ca/bc/gov/open/jagefilingapi/qa/restdata/Payload.java

```diff
@@ -0,0 +1,33 @@
+package ca.bc.gov.open.jagefilingapi.qa.restdata;
+
+public class Payload {
+
+    public static String submitDocumentProperties() {
+        return "{\n" +
+                "  \"documentProperties\": {\n" +
+                "    \"type\": \"string\",\n" +
+                "    \"subType\": \"string\",\n" +
+                "    \"submissionAccess\": {\n" +
+                "      \"url\": \"string\",\n" +
```

Comment:
this is actually a cc suggestion, but yeah we could refactor this to have a variable:

```
String urlSection = \"url\": \"string\",\n";
```

and then use it throughout this large string since it is replicated many times

Reply:
Not applicable anymore. Thanks.